<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Print Reports</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Pages from</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_from" >
       <property name="minimum" >
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_to" >
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="value" >
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QCheckBox" name="checkBox_reverse" >
         <property name="text" >
          <string>last page first</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_pdf" >
         <property name="windowModality" >
          <enum>Qt::WindowModal</enum>
         </property>
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>1</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>print to file (pdf)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_oddEven" >
       <property name="title" >
        <string/>
       </property>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBox_odd" >
          <property name="text" >
           <string>odd pages</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox_even" >
          <property name="text" >
           <string>even pages</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok" >
       <property name="text" >
        <string>OK</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_ok</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>171</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel" >
     <x>149</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>252</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel" >
     <x>149</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
